### ISSUE TITLE
Adding Text Color - a comment, not an issue

### ISSUE THREAD

**Issue by fredmc** [wimbarelds/TimeCircles#95]
I apologize if this should not be posted in the Issues. It is not an issue. It is how I modified the code to change the text color. Black text on dark background didn't work for me.

For what it's worth, I added a config option: text_color, e.g.
`$(".example").TimeCircles({text_color: "#FFFFFF"});`

I modified TimeCircles,js thusly:
In 
`TC_Instance.prototype.default_options = { `
changed
`text_size: 0.07,`
to
```
text_size: 0.07,
text_color: "#000000",
```
In the section // Prepare Time Elements changed
`headerElement.appendTo(textElement);`
to
```
headerElement.css("color", this.config.text_color);
headerElement.appendTo(textElement);
```
changed
`numberElement.appendTo(textElement);`
to
```
numberElement.css("color", this.config.text_color);
numberElement.appendTo(textElement);
```
At first I added css color to textElement, but other css on the page over rode that for h4, the headerElement.

**Comment by wimbarelds** [comment]
Though you can do this, it would be better (I think) to use plain CSS. So:

````css
#timer .time_circles .textDiv_Days {
    color: #F00;
}
````

**Comment by fredmc** [comment]
True. That may have been better. But, the way I did it, I can just make it part of the config without messing with the css files. And I just have to put in the color one place, rather than once for each div. Also, the numberElement is in h4 and unless I change h4 for everything, it doesn’t get that. At least on my page, I added the css color to the just the textDiv_Days (etc.) and the number was still black, even though it’s in that div. Putting it directly on that h4 element did the trick. And yes, I could likely have done that with plain css, 

.textDiv_Days h4

 

Guess it’s a matter of preference or perhaps also how the page is built.

**Comment by wimbarelds** [comment]
You're absolutely right that the rest of the config is done in js; that's definitely something I will be doing different in the next version (if I end up making it). It's a bit sloppy to have the style in the javascript.

**Comment by fredmc** [comment]
imho, I don’t think it’s bad to do it in js. Makes it a ton easier to over ride any other css, without having to use !important all over the place. Like I said, I had to put the color directly on the headerElement (h4) to get it to over ride the h4 definition elsewhere.

**Comment by wimbarelds** [comment]
In the current version; Yeah using CSS probably won't work perfectly. The next version would be (if I make it) a recode from scratch.

**Comment by punitganshani** [comment]
+1 for this feature
